package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@govuk-one-login/frontend-language-toggle",
+  "version": "0.0.5-alpha",
```

Comment:
Does this need to be updated to 1.0.0

Reply:
@Joanneyoung01 thank you , have updated and added the changes from the latest fix